File: app/src/main/java/com/icerrate/popularmovies/provider/cloud/api/ServerPaths.java
package com.icerrate.popularmovies.provider.cloud.api;

/**
 * @author Ivan Cerrate.
 */

public interface ServerPaths {

    String QUERY_PAGE = "page";

    String QUERY_STRING = "query";

    String MOVIE_ID = "id";

    //URLS
    String POPULAR_MOVIES = "/3/movie/popular/";

    String TOP_RATED_MOVIES = "/3/movie/top_rated/";

    String SEARCH_MOVIES = "/3/search/movie";

    String TRAILERS_MOVIES = "/3/movie/{" + MOVIE_ID + "}/videos";

    String REVIEWS = "/3/movie/{" + MOVIE_ID + "}/reviews";
}

File: app/src/main/java/com/icerrate/popularmovies/provider/cloud/api/ServerPaths.kt
package com.icerrate.popularmovies.provider.cloud.api

/**
 * @author Ivan Cerrate.
 */
object ServerPaths {
    const val QUERY_PAGE = "page"
    const val QUERY_STRING = "query"
    const val MOVIE_ID = "id"

    // URLS
    const val POPULAR_MOVIES = "/3/movie/popular"
    const val TOP_RATED_MOVIES = "/3/movie/top_rated"
    const val SEARCH_MOVIES = "/3/search/movie"
    const val TRAILERS_MOVIES = "/3/movie/{$MOVIE_ID}/videos"
    const val REVIEWS = "/3/movie/{$MOVIE_ID}/reviews"
}